### DISCUSSION TITLE
Scanning a range of Ip addresses and specific ports.

### DISCUSSION THREAD

**Discussion by paolosezart** [projectdiscovery/naabu#897] [Q&A]
Please show an example how to scan a specific range of IP addresses and TCP ports.
For example:
192.168.1.1-192.168.1.50 and ports 22, 80, 443, 10000-11000

According to the instructions I understood that I need to take the IP range from a file. But how to set these parameters in the command line.
Thanks.

**Comment by sec13b** [comment]
also i want do something like you ask.

you can try :
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cat ip.txt | sudo  naabu -p 22, 80, 443, 10000-11000 -rate 1 -retries 1 -timeout 500 -s -Pn -sn -pe -debug  -stats -v 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
maybe is what you need

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
